polybar-scripts/info-tmux-sessions/README.md

```diff
@@ -0,0 +1,21 @@
+# Script: info-tmux-sessions
+
+Reports the state of tmux sessions on the local machine.
+
+* with active sessions
+![info-tmux-sessions](screenshots/1.png)
```

Comment:
The font is a bit small, maybe we need better screenshots

Reply:
I thought that might be a concern.

I'll set up a secondary bar and get clearer screenshots tomorrow.

Reply:
Much bigger screenshots now.